Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -404,6 +404,60 @@ for armv7.
 Attaching to process 2670...
 ```
 
+###### Launching an application via debugserver
+
+```shell
+debugserver -x backboard *:1234 /Applications/MobileSMS.app/MobileSMS
+```
+
+Attach to an already running application
```

Comment:
Is this an extra step after -x ? or does -x also attach? 
```suggestion
Attach to an already running application:
```

Reply:
-x is the flag to tell debugserver to launch application